### DISCUSSION TITLE
EDC Extension for Access Management on Submodel Server(s) of Asset Administration Shells from Digital Twin Registry

### DISCUSSION THREAD

**Discussion by thomas-henn** [eclipse-tractusx/tractusx-edc#1062] [Ideas]
As part of the https://github.com/eclipse-tractusx/sig-release/issues/417
an EDC Extension serving as "Policy Enforcement Point" is required.

With this pull request https://github.com/eclipse-tractusx/tractusx-edc/pull/1028
on that we would like to make this EDC Extension as being part of the Tractus-X EDC,
since the Tractus-X Digital Twin Registry and the use with Asset Administration Shells is very much in use with other Tractus-X applications, *e.g.*, 
* https://github.com/eclipse-tractusx/puris-backend
* https://github.com/eclipse-tractusx/demand-capacity-mgmt
* https://github.com/eclipse-tractusx/digital-product-pass
* https://github.com/eclipse-tractusx/managed-simple-data-exchanger-backend
* https://github.com/eclipse-tractusx/item-relationship-service
* https://github.com/eclipse-tractusx/traceability-foss-backend

beside the use cases in Tractus-X (see Kits https://eclipse-tractusx.github.io/Kits).

Thank you very much for your contribution on this discussion.

**Comment by ma3u** [comment]
How we solve this? We need a policy enforcement point in EDC for Digital Twins. 

Can we use the data plane signaling? 
https://github.com/eclipse-edc/Connector/tree/main/docs/developer/decision-records/2023-12-12-dataplane-signaling
https://github.com/eclipse-edc/Connector/blob/main/docs/developer/data-plane-signaling/data-plane-signaling.md

**Comment by lgblaumeiser** [comment]
Closed as stale, please renew item if this is still an issue
